okhttp/src/main/java/okhttp3/internal/Internal.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2014 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal
+
+import okhttp3.ConnectionPool
+import okhttp3.OkHttpClient
+import okhttp3.Response
+import okhttp3.internal.connection.Exchange
+import okhttp3.internal.connection.RealConnectionPool
+
+/**
+ * Escalate internal APIs in `okhttp3` so they can be used from OkHttp's implementation
+ * packages. The only implementation of this interface is in [OkHttpClient].
```

Comment:
Maybe change these docs to `Escalate internal APIs in okhttp3 so they can be used by friend modules.

Reply:
You beat me to it!